source/configuration_manual/fts/squat.rst

```diff
@@ -0,0 +1,150 @@
+.. _fts_backend_squat:
+
+Squat FTS Engine
+================
+
+NOTE: The Squat code is quite slow for large mailboxes. There are also a
+few bugs that are unlikely to be fixed. It's recommended to use
+:ref:`fts_backend_solr` instead.
```

Comment:
please indicate here that SQUAT is obsolete, should not be used and is going to be removed.